src/ExpressionEvaluator/CSharp/Test/ExpressionCompiler/PerfTests/DebuggerSteppingTestcase.xml

```diff
@@ -51,7 +49,6 @@
         <StepOver waitForBreakOrEnd="true"/>
       </Debug>
       <WaitForCodeMarkers />
-      <MeasureTimeStop /><!-- End timer (Not sure if correct command) -->
       <Debug>
         <StopDebugging />
```

Comment:
Should I make a C# specific version and a VB specific version of the test? (Since, by my understanding, these lines would be checking that the VB dlls are being loaded and that the C# ones aren't)